src/main/java/com/dawn/jat/illuminati/comment/repository/CommentRepository.java

```diff
@@ -0,0 +1,4 @@
+package com.dawn.jat.illuminati.comment.repository;
+
```

Comment:
this is just a suggestion, we dont need to add a Comment right now :d

Reply:
Removed